esmvaltool/recipes/recipe_consecdrydays.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+datasets:
+   - {dataset: bcc-csm1-1-m,  project: CMIP5,  mip: day,  exp: historical,  ensemble: r1i1p1,  start_year: 2001,  end_year: 2002}
+
+preprocessors:
+
+   preprocessor1: {}
```

Comment:
There is no need to define a preprocessor if you only use the defaults, you can just leave it out.